controller/controller.go

```diff
@@ -342,23 +361,24 @@ func (c *Impl) EnqueueNamespaceOf(obj interface{}) {
 
 // EnqueueKey takes a namespace/name string and puts it onto the work queue.
 func (c *Impl) EnqueueKey(key types.NamespacedName) {
-	c.WorkQueue.Add(key)
-	c.logger.Debugf("Adding to queue %s (depth: %d)", safeKey(key), c.WorkQueue.Len())
+	c.workQueue.Add(key)
+	c.logger.Debugf("Adding to queue %s (depth: %d)", safeKey(key), c.workQueue.Len())
 }
 
-// MaybeEnqueueBucketKey takes a Bucket and namespace/name string and puts it onto the work queue.
-func (c *Impl) MaybeEnqueueBucketKey(bkt reconciler.Bucket, key types.NamespacedName) {
+// MaybeEnqueueBucketKey takes a Bucket and namespace/name string and puts it onto
+// the slow work queue.
+func (c *Impl) maybeEnqueueBucketKey(bkt reconciler.Bucket, key types.NamespacedName) {
 	if bkt.Has(key) {
-		c.WorkQueue.Add(key)
-		c.logger.Debugf("Adding to queue %s (depth: %d)", safeKey(key), c.WorkQueue.Len())
+		c.workQueue.SlowLane().Add(key)
+		c.logger.Debugf("Adding to queue %s (depth: %d)", safeKey(key), c.workQueue.Len())
```

Comment:
Call EnqueueSlowKey.

Reply:
renamed but kept both in one. No usage of the other one so far.